server.js

```diff
@@ -1,10 +1,23 @@
 const express = require('express')
 const app = express()
+const mongoose = require('mongoose')
+const URI = 'mongodb+srv://Rishabh:qwertyuiop@cluster0.8yyadat.mongodb.net/?retryWrites=true&w=majority'
+mongoose.connect(URI)
+.then(()=>{
+  app.get('/',(req,res)=>{
+    res.json({Connection_Status:'Connected'})
+  })
+}).catch((err)=>{
+  app.get('/',(req,res)=>{
+    res.json({Connection_Status:'Not Conncected'})
+  })
+})
 
-app.get('/', (req, res) => {
+app.get('/ping', (req, res) => {
```

Comment:
Consider using more descriptive endpoint names. '/ping' is often used for health checks and might be confusing in this context. [medium]